### ISSUE TITLE
ignoreProperties does not work

### ISSUE THREAD

**Issue by MoskalenkoM** [stylelint/stylelint#4764]
Sorry for this issue if you have such.
I have a problem with ignoreProperties in stylelint.config.js

module.exports = {
  extends: 'stylelint-config-standard',
  ignoreProperties: ['text-decoration-skip-ink'],
  rules: {}
};

But I have a problem with this
a:not([class]) {
  text-decoration-skip-ink: auto;
}
error output: Unknown property: 'text-decoration-skip-ink' postcss (unknownProperties)

How may fix it?
Thanks.

**Comment by jeddy3** [comment]
`ignoreProperties` is a secondary option for the `property-no-unknown` rule, so your configuration object should look like:

```js
module.exports = {
	extends: 'stylelint-config-standard',
	rules: {
		'property-no-unknown': [true, { ignoreProperties: ['text-decoration-skip-ink'] }],
	},
};
```

[User guide.](https://stylelint.io/user-guide/configure#rules)

---

`text-decoration-skip-ink` is a known property and stylelint won't warn for it (see [this demo](https://stylelint.io/demo#N4Igxg9gJgpiBcICGACYKAuMAeGC0skATkhgJYQB2eAzgNZkAOeZld8KSArhhANwoAvgB1KIADQgaAT0oYk2BOBo0J4KgDMyAcyXBRKFMJBEuAGxg1jHfZUOHjjIhEYwiGaXkoQ8XNt4B3SmtMUxgDIVFBEEEgA)).

Make sure you're using the latest version of stylelint.

> Unknown property: 'text-decoration-skip-ink' postcss (unknownProperties)

This doesn't look like stylelint output. Perhaps you're running two linter programs.

**Comment by MoskalenkoM** [comment]
Yes, I wrote different versions, but yours doesn't work either
